.github/workflows/pr_title_checker_config.json

```
@@ -0,0 +1,11 @@
+{
+  "CHECKS": {
+    "regexp": "^(Revert \")?WT-[0-9]+ ",
+    "alwaysPassCI": false
+  },
+  "MESSAGES": {
+    "success": "All OK",
+    "failure": "Failing CI test",
```

Same comment here about user friendliness
```suggestion
    "failure": "PR title does not match the regex: \"^(Revert \")?WT-[0-9]+ \"",
```